src/routes/wallets/add/+page.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+  import { goto } from '$app/navigation'
+  import { db } from '$lib/db/index.js'
+  import { nowSeconds } from '$lib/utils.js'
+  import { createRandomHex } from '$lib/crypto.js'
+  import { walletTypeToInitialConfiguration } from '$lib/wallets'
+  import { onMount } from 'svelte'
+
+  onMount(async () => {
+    // add new CLN wallet to the db with generic label and random id
+    const id = createRandomHex()
+    const type = 'coreln'
+    const label = type
+
+    await db.wallets.add({
+      id,
+      label,
+      type,
+      createdAt: nowSeconds(),
+      modifiedAt: null,
+      configuration: walletTypeToInitialConfiguration(type),
+      lastSync: null,
+      syncing: false
+    })
+
+    // route to the created wallet id
+    await goto(`/wallets/${id}`)
+  })
```

Comment:
Instead of adding a route to just add the wallet to the db, you could just move this logic in to a function and call it whenever we link/route to `/add` instead. So the flow would be call the `addWallet` function, then just route to the new wallet's `id`.

Reply:
oh yeah that makes sense. Ill have to take a look at how to do this in the anchor tags we have inside translations: 

```
 "get_started_hint": "To get started, add a new <a href='/wallets/add'>wallet</a>."
 ```

Reply:
Yeah I would remove the anchor tag from that text and turn the "wallet" text into a button. The styling can look the same, but the button will have an on:click handler that then calls the `addWallet` function and then routes using `goto` to the wallet route.